src/app/tasks/services/tasks-statuses.service.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { TasksStatusesService } from './tasks-statuses.service';
+
+
+
+describe('tasksStatusesService', () => {
+  const service: TasksStatusesService = new TasksStatusesService();
+
+
+  test('should create TasksStatusesService', () => {
+    expect(service).toBeTruthy();
+  });
+
+  test('should return the right task status', () => {
+    const mockStatus = service.statusToLabel(10);
+    expect(mockStatus).toBe('Processed');
+  });
+  
+
+  test('should return true', () => {
+    const mockRetriedTaskStatus = service.isRetried(11);
+    expect(mockRetriedTaskStatus).toBeTruthy();
+  });
```

Comment:
No need to mock the result. Instead you could just go:
```suggestion
  test('should return true', () => {
    expect(service.isRetried(11)).toBeTruthy();
  });
```

The name `mock` is really of use when we want to fake a service. If we are just storing a value inside, it isn't a mock.